packages/eslint-plugin/src/rules/await-thenable.ts

```diff
@@ -35,6 +38,17 @@ export default util.createRule({
           context.report({
             messageId: 'await',
             node,
+            suggest: [
+              {
+                messageId: 'removeAwait',
+                fix(fixer): TSESLint.RuleFix {
+                  return fixer.removeRange([
+                    node.range[0],
+                    node.range[0] + 'await'.length,
+                  ]);
```

Comment:
should probably go via the token to do this so we're a bit stricter
something like
```ts
const sourceCode = context.getSourceCode();
const awaitKeyword = util.nullThrows(sourceCode.getFirstToken(node, isAwaitKeyword), util.NullThrowsReasons.MissingToken('await', 'await expression'));
return fixer.remove(awaitKeyword);
```

Reply:
Aha yes! I always forget this exists. Thanks.

I wonder if we should audit or even lint for existing rules...